### PR TITLE
Installation script

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

**Comment by PavelBal** [comment]
- Some changes to installation script as proposed here https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/1143/files#diff-aa94d1f3ff4649aaf162eed5d0e0dccefa541a30ccbb3f77075b6e97c0256cf1

- Update vignettes

**Review by svavil — CHANGES_REQUESTED** [review]

File: inst/extdata/install-packages.R [hunk 1]
```diff
@@ -117,46 +114,43 @@ displayProgress <- function(current, success = TRUE, message = NULL, suppressOut
...
+installOSPPackages <- function(rclrVersion = "0.9.2",
                                suppressOutput = TRUE,
-                               developerVersion = FALSE){
-  # Install dependencies from CRAN
-  displayProgress("Installing CRAN packages", suppressOutput = suppressOutput)
-  install.packages(.cranPackages,
-                   dependencies = TRUE)
-
+                               developerVersion = FALSE,
+                               lib = NULL){
```
[the remark this answers is not in the document]
Comment (PavelBal) [reply inst/extdata/install-packages.R:132]: Don't know, is it always the user library? And how will it behave when installing to renv?

[the remark this answers is not in the document]
Comment (svavil) [reply inst/extdata/install-packages.R:132]: Your parameter documentation says `lib` defaults to `libPaths()[1]` if it is missing, but is set to `NULL` in the function signature. I am not sure it makes a difference, though

[the remark this answers is not in the document]
Comment (PavelBal) [reply inst/extdata/install-packages.R]: I changed the defaults.

**Review by svavil — APPROVED** [review]

File: vignettes/data-handling.Rmd [hunk 2]
```diff
@@ -27,19 +27,27 @@ library(esqlabsR)
 
 The workflow of modeling and simulation involves running simulations and comparing simulated data against observed data. Functionalities of `esqlabsR` require observed data to be present as [`DataSet`](https://www.open-systems-pharmacology.org/OSPSuite-R/reference/DataSet.html) objects. Please refer to the article [Observed data](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/observed-data.html) for information on how to load data from Excel or `*.pkml` files.
```
Comment (hannaei) [inline vignettes/data-handling.Rmd:28]: Is the link here supposed to be [https://www.open-systems-pharmacology.org/OSPSuite-R/dev/articles/observed-data.html](url)?

Comment (PavelBal) [reply vignettes/data-handling.Rmd:28]: True, will fix.

**Pull request merged**